data/saas/employees_hiring-agent.yaml

```
@@ -0,0 +1,65 @@
+- id: employees_hiring-agent
+  name: トム
+  role: 採用担当（Recruiter）
+  description: 採用活動の設計と運用（求人票テンプレ、候補者DB、面談スケジュール、同意管理）。従業員カードとダッシュボードにタスクを可視化します。
+  
+  responsible_files:
+    - kamuios/data/saas/prompts-repo.yaml
+    - static/data/prompts/hiring-agent.md
+    - static/videos/recruiter_card_video_square.mp4
+    - static/images/agent_card.png
+    - themes/kamui-docs/static/js/main.js
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)